.ci/dockerfiles/Dockerfile.xpu

```
@@ -0,0 +1,134 @@
+        linux-libc-dev \
+        libaio-dev && \
+    # Install Intel GPU runtime packages
+    apt-get install -y intel-oneapi-dpcpp-ct=${ONEAPI_CT_VERSION} && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Additional libs to support RDMA over Ethernet (RoCE) / Infiniband
+RUN apt-get update -y && \
+    apt-get install -y librdmacm-dev ibverbs-utils libibverbs-dev libibverbs1 librdmacm1 rdma-core rdmacm-utils && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update -y && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)